java-rmi/src/main/java/com/baeldung/rmi/RMIClientImpl.java

```diff
@@ -0,0 +1,23 @@
+package com.baeldung.rmi;
+
+import java.rmi.NotBoundException;
+import java.rmi.RemoteException;
+import java.rmi.registry.LocateRegistry;
+import java.rmi.registry.Registry;
+
+public class RMIClientImpl {
```

Comment:
I thought we were going to delete this? Let's go ahead and do that. I don't think it adds any value over and above your integration test.

Reply:
I was waiting for your input, you did not reply so it was not deleted. I will do that.

You are yet to also give clarity on the system.out and similar code rules.

Reply:
Thanks, Gautam. Personally, a standalone class that doesn't have an output doesn't have value--I think you would be safe to have a sysout, so long as it is just in the repo and not in the article.